tutorials/counter/README.md

```diff
@@ -0,0 +1,227 @@
+<div id="top"></div>
+
+<div align="center">
+  <a href="https://github.com/FuelLabs/sway-applications">
+    <img src="./img/logo.png" height="120">
+  </a>
+
+  <h3 align="center">Sway Counter</h3>
+
+  <p align="center">
+    Sway Counter Smart Contract with React and Fuel v2 TypeScript SDK
+    <br />
+    <a  href="https://fuellabs.github.io/fuels-ts/"><strong>Explore the Fuel TypeScript SDK docs »</strong></a>
+  </p>
+</div>
+
+## Table of contents
+
+- [About](#about)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [How to run the application](#usage)
+  - [Application Development Tutorial](#walkthrough)
+    - [Creating the smart contract](#1-creating-a-deployable-sway-program)
+    - Compiling the contract, generating TypeScript and deploying
+      - [Compiling the contract](#2-compiling-your-sway-contract)
+      - [Generating TypeScript](#3-generating-typescript-for-sway-contract)
+      - [Deploying the contract](#4-deploying-your-sway-contract)
+    - [Using the contract in a React project](#5-using-your-sway-contract-in-a-react-project)
+- [Contributing](#contributing)
+- [License](#license)
+
+## About
+
+In this tutorial, we will build a counter application, using storage on our Sway contract to keep track of a stored number that can be retrieved/ modified as part of our React app. This demonstrates a Sway Program called Smart Contract, and will show you the basics of using Sway with the [TypeScript SDK](https://fuellabs.github.io/fuels-ts/).
+
+<p align="center">
+  ![Demo Screen Shot](./img/preview.png)
+</p>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Getting Started
+
+We recommend that you first read through and understand the basics of [Sway](https://fuellabs.github.io/sway/latest/), the DSL for the Fuel Virtual Machine.
+
+### Prerequisites
+
+Your machine will need to have a few things installed in order to run a local fuel node and this React application:
+
+- [The latest LTS version of Node.js](https://nodejs.org/)
+- [The latest forc and fuel-core binaries](https://fuellabs.github.io/sway/latest/introduction/installation.html#dependencies)
+
+### Installation
+
+Open this demo application
+
+```sh
+cd tutorials/counter
+```
+
+Install NPM packages
+
+```sh
+npm i
+```
+
+### Usage
+
+To quickly run this demo application, follow these steps, or see detailed tutorial below
+
+Run React application in dev mode, make sure to configure a new [`.env`](./.env.example) file.
+
+```sh
+npm run dev
+```
+
+View in local browser
+
+```sh
```

Comment:
Is this a command? If this is not a command then it should not be in a codeblock and instead it should be a regular sentence

Reply:
Really? `open` is not a default command on Arch Linux or on Ubuntu.